build-tools/Xamarin.Android.Tools.BootstrapTasks/Xamarin.Android.Tools.BootstrapTasks/CheckApiCompatibility.cs

```diff
@@ -268,6 +274,7 @@ void ValidateIssues (StreamReader content, bool validateAgainstReference)
 				if (currentSet == null) {
 					// Hashset should never be null, unless exception file is not defining assembly line.
 					Log.LogError ($"Exception report/file should start with: Compat issues with assembly");
```

Comment:
Perhaps the `LogError()` itself should be updated?

```csharp
Log.LogError ($"Exception report/file should start with: `Compat issues with assembly`; was: `{line}`");
```

Reply:
Great suggestion! Fixed.